text/symbols.talon

```diff
@@ -62,7 +65,10 @@ angle that:
 percent that:
     text = edit.selected_text()
     user.paste("%{text}%")
-quote that:
+(quote | string) that:
```

Comment:
`string that` is already a command, since `string` is a formatter and `<user.formatters> that` is a command. so probably just make this "quote that".

Reply:
Good call, thanks

Reply:
Okay, amended